src/main/java/org/opensrp/repository/postgres/mapper/custom/xml/CustomStockMetadataMapper.xml

```diff
@@ -140,4 +153,23 @@
         	where stock_id = #{stockId,jdbcType=BIGINT}
     </select>
 
+	<select id="selectManyStockAndProductDetailsByServicePointId" resultMap="StockAndProductDetails">
```

Comment:
Here is the custom query responsible to fetch both stock and product catalog details @githengi